src/controllers/PermissionController.js

```diff
@@ -0,0 +1,35 @@
+import dotenv from 'dotenv';
+import { User } from '../queries';
+import status from '../config/status';
+
+dotenv.config();
```

Comment:
I can't see where the dotenv.configs are being invoked

Reply:
Changed applied